include/InstrumentTrackWindow.h

```diff
@@ -148,8 +148,8 @@ protected slots:
 	Knob * m_volumeKnob;
 	Knob * m_panningKnob;
 	Knob * m_pitchKnob;
-	PixmapButton *m_muteBtn;
-	PixmapButton *m_soloBtn;
+	AutomatableButton *m_muteBtn;
+	AutomatableButton *m_soloBtn;
```

Comment:
While I don't think we've officially added it to the coding convention document yet, there's been strong agreement that we should put the `*` on the left side. (See [here](https://discord.com/channels/203559236729438208/207159523616489473/785877041132208148))

It makes things less context-dependent and easier to visually parse. For example:
```cpp
foo* bar   // always pointer
foo * bar  // always multiplication
foo *bar   // always pointer dereference
foo& bar   // always reference
foo & bar  // always bitwise AND
foo &bar   // always address of
```

Reply:
> It makes things less context-dependent

The thing is, the symbol *is* inherently context-dependent. It does appear to be mentioned in that thread as well: `int* a, b` is equivalent to `int *a; int b` (though "How often do you need to declare multiple pointers on one line?" is a perfectly valid criticism). It's cursed (thankfully newer programming languages don't do this) but it is the truth regarding C and C++ pointers.

Regardless of my personal preference or how the language syntax is defined, I'll happily conform to our coding conventions, if/when this is officially added to them. Until then, the [Qt Coding Style](https://wiki.qt.io/Qt_Coding_Style#Spaces_within_code) *do* specify otherwise, which our coding conventions "may" fall back to when we are missing something.

Reply:
I added `*` since the Discord vote was like 11:3 in favor of this style. Please modify as needed: https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers

Reply:
Now that it's official, I'll make those changes here and start doing that moving forward.